src/backend/catalog/ag_label.c

```diff
@@ -188,8 +188,9 @@ Datum _label_name(PG_FUNCTION_ARGS)
     uint32 label_id;
 
     if (PG_ARGISNULL(0) || PG_ARGISNULL(1))
-        ereport(ERROR, (errcode(ERRCODE_NULL_VALUE_NOT_ALLOWED),
```

Comment:
This error message should not be changed. It hides an issue by having it return NULL.